hw3/output/Words.group_FDR.md

```diff
@@ -0,0 +1,142 @@
+# Words-Fixation Activation Map (FDR Correction)
+* q = .05 
+* coordinates are center of mass of each cluster in MNI space  
+ 
+| Cluster Size | Peak Value (Z) | x (mm) | y (mm) | z (mm) |
```

Comment:
Note that these are corrected z values. You would have to run 3dclust on the uncorrected z values, masked by the thresholded FDR-corrected Z map to get uncorrected Z values.